.github/workflows/docker.yml

```
@@ -43,14 +43,18 @@ jobs:
             VERSION: ${{ env.VERSION }}
             VERSION_SUFFIX: ${{ env.VERSION_SUFFIX }}
     build-docker-single-arch:
-        name: build-docker-${{ matrix.binary }}
+        name: build-docker-${{ matrix.binary }}-${{ matrix.features.version_suffix }}
         runs-on: ubuntu-22.04
         strategy:
             matrix:
                 binary: [aarch64,
                          aarch64-portable,
                          x86_64,
                          x86_64-portable]
+                features: [
+                    {version_suffix: "", env: ""},
+                    {version_suffix: "-minimal", env: "spec-minimal"}
```

```suggestion
                    {version_suffix: "-dev", env: "spec-minimal"}
```

I was thinking that `-minimal` might attract users since they might think the image is smaller or something. I'm thinking `-dev` looks a little less appealing. It would also give us the option to add some more features in this build, if we saw fit.

I'm open to other naming suggestions, too!